easyaudit/migrations/0018_auto_20240105_0125.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 3.2 on 2024-01-05 01:25
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('easyaudit', '0017_alter_requestevent_datetime'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
What are all the `AlterField` operations caused by?

Reply:
Many of these fields allowed `null=True, blank=True`, which is bad. That means there are two possible values with the same meaning. This rule explains it well:

https://docs.astral.sh/ruff/rules/django-nullable-model-string-field/

I changed all of them to `blank=True, default=""`, which will not propagate `None` as a value, but still allow the field to be blank.